src/middleware/frontend-log-middleware.ts

```diff
@@ -65,3 +66,20 @@ export const LogMiddleware = async (req: Request, res: Response, next: NextFunct
 	});
 	next();
 };
+
+const getUnvalidatedJiraHost = (req: Request): string | undefined =>
+	req.session?.jiraHost || extractUnsafeJiraHost(req);
+
+const checkPathValidity = (url: string) => moduleUrls.some(moduleUrl => matchRouteWithPattern(moduleUrl, url));
+
+const extractUnsafeJiraHost = (req: Request): string | undefined => {
```

Comment:
That's for logging only, right?

I think we could change it to something like
```
const createJiraHostChildLogger = (logger: Logger) => {
    ....

}
```

and encapsulate all the related to jiraHost logic here, removing all confusing words like "validated/unsafe/validity" etc. Wdyt?

Reply:
This file doesn't require any change. Changes reverted.